src/components/ciConfig/CIConfig.tsx

```diff
@@ -168,76 +179,191 @@ function Form({ dockerRegistries, sourceConfig, ciConfig, reload, appId }) {
+                            }
+                        }}
+                        components={{
+                            IndicatorSeparator: null,
+                            Option: (props) => {
+                                return <components.Option {...props}>
+                                    <div style={{display: 'flex'}}>
+                                    {props.isSelected ? <Check className="icon-dim-16 vertical-align-middle scb-5 mr-8 mt-4" /> : <span className="inline-block icon-dim-16 mr-8"></span>}
+                                    <div className={'registry-icon mr-5 ' + props.data.registryType}></div>
+                                    {props.label}
+                                    </div>
+                                </components.Option>
+                            },
+                            MenuList: (props) => {
+                              return <components.MenuList {...props}>
+                                  {props.children}
+                                  <NavLink to={`${URLS.GLOBAL_CONFIG_DOCKER}`} className="cb-5 select__sticky-bottom block fw-5 anchor w-100 cursor no-decor" style={{backgroundColor: '#FFF'}}>
                                 <Add className="icon-dim-20 mr-5 fcb-5 mr-12 vertical-align-bottom" />
                                 Add Container Registry
                             </NavLink>
-                        </Select>
+                              </components.MenuList>
+                          },
+                            Control: (props) => {
+                                let value = "";
+                                if (props.hasValue) {
+                                    value = props.getValue()[0].registryType;
+                                }
+                                return <components.Control {...props}>
+                                    <div className={'registry-icon ml-5 ' + value}></div>
+                                    {props.children}
+                                </components.Control>
+                            },
+                        }}
+                        onChange={(selected) => { handleRegistryChange(selected) }}
+                    />
                         {registry.error && <label className="form__error">{registry.error}</label>}
                     </div>
                     <div className="form__field">
-                        <label htmlFor="" className="form__label">Container Repository</label>
+                        <label htmlFor="" className="form__label">Container Repository {REGISTRY_TYPE_MAP[selectedRegistry.registryType]?.desiredFormat}</label>
                         <input
                             tabIndex={4}
                             type="text"
                             className="form__input"
-                            placeholder="Enter repository name"
+                            placeholder={REGISTRY_TYPE_MAP[selectedRegistry.registryType]?.placeholderText}
```

Comment:
There should be some default placeholder text if [.......registryType]?.placeholderText is empty or undefined.